runtime/vm/jnicgen.c

```diff
@@ -138,7 +179,7 @@ jdouble JNICALL callVirtualDoubleMethod (JNIEnv *env, jobject receiver, jmethodI
 {
 	va_list va;
 	va_start(va, methodID);
-	RUN_CALLIN_METHOD(env, receiver, NULL, methodID, VA_PTR(va));
+	RUN_CALLIN_METHOD(env, receiver, NULL, methodID, (U_8*)(VA_PTR(va)) + 2);
```

Comment:
Is this supposed to be modified?

Reply:
Fixed under: e4ba6504f90ab6bedab07f23600fef8cc59548fa  Thanks!